ansible_ai_connect/ai/api/wca/tests/test_model_id_views.py

```diff
@@ -276,6 +277,47 @@ def test_set_model_id_throws_validation_exception(self, *args):
             self.assertEqual(r.status_code, HTTPStatus.BAD_REQUEST)
             self.assert_segment_log(log, "modelIdSet", "ValidationError")
 
+    @override_settings(SEGMENT_WRITE_KEY="DUMMY_KEY_VALUE")
+    def test_set_model_id_empty_response(self, *args):
+        self.user.organization = Organization.objects.get_or_create(id=123)[0]
+        mock_secret_manager = apps.get_app_config("ai").get_wca_secret_manager()
+        mock_wca_client = apps.get_app_config("ai").model_mesh_client
+        self.client.force_authenticate(user=self.user)
+
+        # Set ModelId
+        mock_secret_manager.get_secret.return_value = {"SecretString": "someAPIKey"}
+        mock_wca_client.infer_from_parameters = Mock(side_effect=WcaEmptyResponse)
+        with self.assertLogs(logger="ansible_ai_connect.users.signals", level="DEBUG") as signals:
+            with self.assertLogs(logger="root", level="DEBUG") as log:
```

Comment:
I don't really see the point of checking the log, it's already done multiple times in other tests and it reduces the readability of the test.

Reply:
Fair enough. Hasty copy-and-paste.

I can change if we agree this approach is reasonable..  @jameswnl didn't think it desirable so this PR is to gather opinion.